classes/02class/exercises/c02-network08/mfreitassm/README.md

```diff
@@ -0,0 +1,91 @@
+# c02-network08
+
+## Commands Execution Output
+
+- Answer the following questions
+  - What is the difference between inbound x outbound?
+    ```
+    Inbound means incoming traffic coming to your EC2 instances. For that you have to add inbound rule. 
+    Outbound means outgoing traffic from your EC2 instances. To connect internet or any browser you have to add outbound rule.
+    ```
+    
+  - What is going to happen by default? Everything allowed or denied?
+    ```
+   	 When you create a security group, it has no inbound rules. Therefore, no inbound traffic originating from another host to your instance is allowed until you add inbound rules to the security group.
+
+	By default, a security group includes an outbound rule that allows all outbound traffic. You can remove the rule and add outbound rules that allow specific outbound traffic only. If your security group has no outbound rules, no outbound traffic originating from your instance is allowed.
```

Comment:
Can you please review this answer? This question is about NACLs not about SGs, everything else looks good though.